python_examples/basic_infection_history_simulation/triple_simulation_outputs/output.csv

```diff
@@ -0,0 +1,22 @@
+time,InfectionStatus.Susceptible,InfectionStatus.Exposed,InfectionStatus.InfectASympt,InfectionStatus.InfectMild,InfectionStatus.InfectGP,InfectionStatus.InfectHosp,InfectionStatus.InfectICU,InfectionStatus.InfectICURecov,InfectionStatus.Recovered,InfectionStatus.Dead,InfectionStatus.Vaccinated


```

Comment:
Ditto to previous point - what do these simulation results tell us? I thought they were just for our debugging, but if they have wider interest then we need to document that

Reply:
See prev comment (mistake)